website/content/docs/connect/ca/vault.mdx

```diff
@@ -210,30 +210,51 @@ The following Vault policy allows Consul to use pre-existing PKI paths in Vault.
 Consul is granted read-only access to the PKI mount points and the Root CA, but is
 granted full control of the Intermediate or Leaf CA for Connect clients.
 
-In this example the `RootPKIPath` is `connect_root` and the `IntermediatePKIPath`
-is `connect_inter`. These values should be updated for your environment.
+This example uses placeholder values for `RootPKIPath` and `IntermediatePKIPath`.
+Please update those placeholders to match the path values in the CA provider configuration.
 
 <CodeBlockConfig filename="vault-managed-pki-policy.hcl">
 
 ```hcl
-# Existing PKI Mounts
+# -----------------------------------------------------------------------------
```

Comment:
Note to self: I'm still missing the guidance on what permissions are needed to change the provider away from the Vault CA provider.

That should probably be mentioned in the CLI/HTTP API docs as well... I'll address that in a future commit.